CMakeLists.txt

```diff
@@ -7,15 +7,34 @@ option(QHOTKEY_INSTALL "Enable install rule" ON)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(Qt5 COMPONENTS Core Widgets REQUIRED)
+set(MINIMUM_REQUIRED_QT_MAJOR 5)
```

Comment:
I would suggest to just put option: `BUILD_WITH_QT6` boolean option for this. Check how it done in Strawberry: https://github.com/strawberrymusicplayer/strawberry/blob/4636cc74dcfecd8d0160c4814e52b192f566f615/CMakeLists.txt#L143

Reply:
I also had this way in sight.

The downside is that this not sustainable in the long run. Because when we get a new major Qt version (yes, it's really far away), we will need to change it to `WITH_QT7` or so. Having a string specified will require less adjustments in CMake code in the future.

But as you wish :)

Please confirm if you want me to push this change.

Reply:
Sounds reasonable to me!